cli/ballerina-cli/src/main/java/io/ballerina/cli/task/RunNativeImageTestTask.java

```diff
@@ -180,6 +180,14 @@ public void execute(Project project) {
         }
 
         TestUtils.writeToTestSuiteJson(testSuiteMap, testsCachePath);
+        Path nativeConfigPath = null;
+        try {
+            nativeConfigPath = target.getNativeConfigPath();
```

Comment:
Any reason why we are resolving the nativeConfigPath here? Its already resolved in Ln307 previously. Seems like the `createReflectionConfig` has just been moved up a few lines. 

Does it have an impact?

Reply:
There is no specific reason for moving. I just wanted to use the `testSuiteMap` with that method.